src/shims/posix/linux/sync.rs

```diff
@@ -65,6 +65,7 @@ pub fn futex<'tcx>(
                         return Ok(());
                     }
                 };
+                this.check_no_isolation("FUTEX_WAIT with timeout")?;
```

Comment:
Could you remove the backtics from the message in `fn isolation_error`? They will look strange with this, and the other users are fine without the backtics as well I think.